packages/asl-toolbox/scripts/lib/config.js

```diff
@@ -1,3 +1,20 @@
+import { config as loadEnv } from 'dotenv';
+import path from 'path';
+import { fileURLToPath } from 'url';
+
+// Get the current file's directory
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+
+// Check if required environment variables are already set
+if (!process.env.DATA_DB_NAME || !process.env.TASKFLOW_DB_NAME) {
+  // Resolve the path to .env.dist
+  const envPath = path.resolve(__dirname, '../../.env.dist');
```

Comment:
```suggestion
  const envPath = path.resolve(__dirname, '../../.env');
```

The convention is:
* `.env` is the file used to store environment variables, but added to .gitignore as often sensitive values need to be included, so this prevents accidental leaking of secrets if it's ignored by default.
* `.env.dist` is a template file that holds non-sensitive environment variables and placeholderts for sensitive ones, and is committed to the repo

There's a one off cost of devs needing to copy and populate the `.env.dist` template file to `.env` when they first check out the repo, but this is justified as it makes it much less likely that api keys and other secrets get accidentally shared.